descqa/QuickBkgTest.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import unicode_literals, absolute_import, division
+import os
+import sqlite3
+import numpy as np
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+
+__all__ = ['QuickBkgTest']
+
+
+def compute_bkg(image):
+    """
+    Routine to give an estimate of the mean, median and std
+    of the background level from  a given image
+
+    Args:
+    -----
+    image : np.array
+
+    Returns:
+    --------
+    mean_bkg : Mean background level
+    median_bkg : Median background level
+    bkg_noise: Background noise level
+    """
+    image = image.flatten()
+    q95 = np.percentile(image, 95)  # This is kind of arbitrary but it works fine
+    q5 = np.percentile(image, 5)  # Same as above -> can be substituted by 10 and 90
```

Comment:
Lines 27-28: combine them. Also, since the comment suggests these numbers can be changed, do not code the number into variable name
```python
q_low, q_high = np.percentile(image, [5, 95])
```